packages/components/src/Loading/LoadingSpinner/_docs/LoadingSpinner.stories.tsx

```diff
@@ -25,12 +25,40 @@ export const Playground: Story = {
 }
 
 export const ClassNameOverride: Story = {
-  args: { classNameOverride: "text-green-400" },
+  render: args => (
+    <LoadingSpinner {...args} classNameOverride="text-green-400" />
+  ),
+  decorators: [
+    Story => (
+      <div className="flex gap-24">
+        <Story />
+      </div>
+    ),
+  ],
```

Comment:
Less lines if we revert 😉 
```suggestion
args: { classNameOverride: "text-green-400" },
```

Reply:
Now that's usin' the ol' noggin 👍  will do